packages/components/src/components/install-guide/record-instructions/IntelliJ.vue

```diff
@@ -1,65 +1,84 @@
-      </p>
-      <p>
-        For more information, visit
-        <a
-          href="https://appmap.io/docs/reference/jetbrains.html#create-appmaps-from-junit-test-runs"
-          target="_blank"
-          >AppMap docs - IntelliJ Tests recording</a
-        >.
-      </p>
-    </template>
-    <template v-else>
-      <VTestsPrompt frameworks="JUnit" />
+      <section class="recording-method">
+        <h3>
+          <i class="header-icon"><TestsIcon /></i>Tests recording<span class="recommended-badge"
+            >recommended</span
+          >
+        </h3>
+        <p>
+          When you run your JUnit tests with the AppMap <code class="inline">javaagent</code> JVM
+          argument, an AppMap will be created for each test.
+        </p>
+        <p>
+          Right-click on any test class or package, and choose
+          <component :is="runConfigIcon" class="run-config-icon" /> "Start with AppMap".
+        </p>
+        <p>
+          For more information, visit
+          <a
+            href="https://appmap.io/docs/reference/jetbrains.html#create-appmaps-from-junit-test-runs"
+            target="_blank"
+            >AppMap docs - IntelliJ Tests recording</a
+          >.
+        </p>
+      </section>
+      <br />
     </template>
+    <template v-else> <VTestsPrompt frameworks="JUnit" /><br /> </template>
     <template v-if="webFramework">
-      <h3>Remote recording</h3>
-      <p>
-        When your application uses {{ this.webFramework.name }}, and you run your application with
-        the AppMap <tt>javaagent</tt> JVM argument, remote recording is enabled. To make a remote
-        recording, run your application using
-        <component :is="runConfigIcon" class="run-config-icon" /> "Start with AppMap". Then use the
-        "Record" button to start recording. Interact with your application, through its user
-        interface and/or by making API requests using a tool such as Postman. When you are done,
-        click the "Record" button again to stop the recording and view the AppMap.
-      </p>
-      <p>
-        For more information, visit
-        <a
-          href="https://appmap.io/docs/reference/jetbrains.html#running-a-java-application-with-appmap"
-          target="_blank"
-          >AppMap docs - IntelliJ Remote recording</a
-        >.
-      </p>
+      <section class="recording-method">
+        <h3>
+          <i class="header-icon"><RemoteRecordingIcon /></i>Remote recording
+        </h3>
+        <p>
+          When your application uses {{ this.webFramework.name }}, and you run your application with
+          the AppMap <code class="inline">javaagent</code> JVM argument, remote recording is
+          enabled. To make a remote recording, run your application using
+          <component :is="runConfigIcon" class="run-config-icon" /> "Start with AppMap". Then use
+          the "Record" button to start recording. Interact with your application, through its user
```

Comment:
This will definitely need a gif of the record button, since that button is just a circle and doesn't even have a hover treatment that says "Record" :(

Reply:
I added this screenshot... gif seemed like overkill. 
<img width="999" alt="image" src="https://github.com/getappmap/appmap-js/assets/1229326/8417c198-bea5-405e-ab4d-6fdc7df2b2fb">

Reply:
@kgilpin heads up that I tweaked the language in the first sentence to describe 'run with AppMap' as a menu item instead of a button.

Reply:
Note that for tests it’s different. You right click on the tests package - afaik